tests/templatetags/test_tags.py

```diff
@@ -4,11 +4,18 @@
 
 register = template.Library()
 
-
+# Basic template tag
 @register.simple_tag
 def error_tag(arg=None):
     "Just raise an exception, never do anything"
     raise Exception("error_tag raised an exception")
 
+# Template tag to to test setting a default.
+@register.simple_tag()
+def default_tag(arg=None):
+    "Just raise an exception, never do anything"
+    raise Exception("default_tag raised an exception")
+
 
+override_tag(register, 'default_tag', default="https://potato.com")
```

Comment:
Some tests for falsey/null defaults would be great.

Reply:
Sure!